.gitlab/e2e.yml

```
@@ -57,6 +57,7 @@ pupernetes-master:
   allow_failure: true # temporary while investigating
   rules:
     - <<: *if_master_branch
+      when: manual # temporary while investigating
```

Nit:
```suggestion
  rules:
    - <<: *if_master_branch
      when: manual # temporary while investigating
      allow_failure: true
```

To harmonize the use of `rules` and define the job behavior in a single place.